src/checkbox/Checkbox.tsx

```diff
@@ -1,61 +1,51 @@
-import React, { FC, memo } from 'react'
+import { FC } from 'react'
 import MuiCheckbox, {
   CheckboxProps as MuiCheckboxProps,
 } from '@mui/material/Checkbox'
 import FormControlLabel from '@mui/material/FormControlLabel'
+import clsx from 'clsx'
 import { Typography } from '../typography'
 
 export type CheckboxProps = MuiCheckboxProps & {
   label?: string
   nopadding?: boolean
 }
 
-const _Checkbox: FC<CheckboxProps> = memo(({ nopadding = false, ...rest }) => (
+const _Checkbox: FC<CheckboxProps> = ({ nopadding = false, ...rest }) => (
   <MuiCheckbox
-    // sx={{ '& .MuiSvgIcon-root': { fontSize: 20 } }}
-    // css={[
-    //   css`
-    //     &.MuiCheckbox-root {
-    //       ${tw`(text-primary-dark-default)!`}
-    //       ${nopadding && tw`p-0!`}
-
-    //       color: ${theme`textColor.shade.medium.default`} !important;
-
-    //       &.Mui-checked {
-    //         color: ${theme`backgroundColor.secondary.dark.default`} !important;
-    //       }
-
-    //       &.Mui-disabled > span path {
-    //         ${tw`text-shade-dark-disabled!`}
-    //       }
-    //     }
-    //   `,
-    // ]}
+    sx={{ '& .MuiSvgIcon-root': { fontSize: 20 } }}
+    classes={{
+      root: clsx(['text-shade-medium-default', nopadding && 'p-0']),
```

Comment:
paddingありの場合は `p-1` でお願いしたいです 🙇 

```suggestion
      root: clsx(['text-shade-medium-default', nopadding ? 'p-0' : 'p-1']),
```